packages/venia-concept/src/index.js

```diff
@@ -110,10 +127,25 @@ const apolloLink = ApolloLink.from([
     Adapter.apolloLink(apiBase)
 ]);
 
+const onIntlError = error => {
+    if (error.code === 'MISSING_TRANSLATION') {
+        console.warn('Missing translation', error.message);
```

Comment:
Does `error.message` include sufficient information here? or do we need to supplement it? :)

Reply:
Error messaging is pretty clear:
`Missing translation [React Intl Error MISSING_TRANSLATION] Missing message: "About Us" for locale "fr-FR", using id as fallback`